packages/legend-application-studio/src/components/editor/edit-panel/connection-editor/RelationalDatabaseConnectionEditor.tsx

```diff
@@ -1151,6 +1135,218 @@ const RelationalConnectionStoreEditor = observer(
+            }
+          }
+        }
+
+        connectionValueState.postProcessorState?.setPostProcessorState(
+          connectionValueState.connection.postProcessors[
+            connectionValueState.connection.postProcessors.length - 1
+          ],
+        );
+      };
+
+    const selectPostProcessor = (postProcessor: PostProcessor): void => {
+      connectionValueState.selectPostProcessor(postProcessor);
+    };
+
+    return (
+      <div className="panel__container ">
+        <ResizablePanelGroup orientation="horizontal">
+          <ResizablePanelSplitter />
+          <ResizablePanel>
+            <div className="panel__container panel__container--border-top">
+              <ResizablePanelGroup orientation="vertical">
+                <ResizablePanel size={150} minSize={70}>
+                  <div className="panel__container panel__container--border-right">
+                    <DropdownMenu
+                      content={postProcessorOptions.map((postProcessorType) => (
+                        <MenuContentItem
+                          key={postProcessorType.value}
+                          onClick={addPostProcessor(postProcessorType.value)}
+                        >
+                          New {postProcessorType.label} Post Processor
+                        </MenuContentItem>
+                      ))}
+                      menuProps={{
+                        anchorOrigin: {
+                          vertical: 'bottom',
+                          horizontal: 'right',
+                        },
+                        transformOrigin: {
+                          vertical: 'top',
+                          horizontal: 'right',
+                        },
+                        elevation: 7,
+                      }}
+                    >
+                      <PanelHeader title="post-processor">
+                        <PanelHeaderActionItem tip="Create Post-Processor">
+                          <PlusIcon />
+                        </PanelHeaderActionItem>
+                      </PanelHeader>
+                    </DropdownMenu>
+                    <div className="panel__content">
+                      {postProcessors.map((postProcessor, idx) => (
+                        <ContextMenu
+                          key={postProcessor._UUID}
+                          disabled={false}
+                          content={
+                            <MenuContent>
+                              <MenuContentItem
+                                onClick={deletePostProcessor(postProcessor)}
+                              >
+                                Delete
+                              </MenuContentItem>
+                            </MenuContent>
+                          }
+                          menuProps={{ elevation: 7 }}
+                        >
+                          <PanelExplorerItem
+                            title={`Post-Processor ${idx + 1}`}
```

Comment:
Probably don't need a title here, is this prop optional for `PanelExplorerItem`, else, let's make it optional?

Reply:
We can make it optional, but I think we need some sort of title for users to differentiate between choosing the post processors? Otherwise it appears likeso: 
<img width="198" alt="image" src="https://user-images.githubusercontent.com/41248956/189752445-d2d7f136-3483-4f57-9367-629d9b60114b.png">

Reply:
Oh, I see, I was thinking about `title` as in tooltip, resolved!